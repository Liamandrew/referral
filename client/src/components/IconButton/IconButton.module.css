.iconButton {
    border: none;
    background-color: transparent;
}

.iconButton > svg {
    fill: #868383;
    width: 24px;
    height: 24px;
}
