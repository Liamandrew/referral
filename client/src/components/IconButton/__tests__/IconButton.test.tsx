import React from "react";
import { render } from "@testing-library/react";
import { ReactComponent as CreateIcon } from "../../../assets/create-24px.svg";
import { IconButton } from "../IconButton";

describe("<IconButton>", () => {
    test("should match the snapshot", () => {
        const { container } = render(
            <IconButton>
                <CreateIcon />
            </IconButton>
        );

        expect(container).toMatchSnapshot();
    });
});
