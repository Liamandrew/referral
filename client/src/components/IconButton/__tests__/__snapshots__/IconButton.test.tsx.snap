// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconButton> should match the snapshot 1`] = `
<div>
  <button
    class="iconButton"
  >
    <svg>
      create-24px.svg
    </svg>
  </button>
</div>
`;
